acceptance/lib/helper.rb

```diff
@@ -316,7 +316,46 @@ def hup_server(host = master, timeout = 30)
       timeout = timeout - sleeptime
     end
   end
-  
+
+  # Update a single settig in trapperkeeper config file using hocon
+  # and the Puppet DSL.
+  # Intent is that this can be used as is or as a building block in
+  # other helpers
+  #
+  # On the server specified in hocon_host, which defaults to master,
+  # update the trapperkeeper config file specified in hocon_file_path,
+  # adding/changing the setting specified in hocon_setting to the value
+  # specified in hocon_value
+  def apply_one_hocon_setting(hocon_host,
+                              hocon_file_path, 
+                              hocon_setting,
+                              hocon_value)
+    hocon_manifest =<<-EOF
+      hocon_setting { "#{hocon_setting}" :
+        ensure  => present,
+        path => "#{hocon_file_path}",
+        setting => "#{hocon_setting}",
+        value => #{hocon_value},
+      }
+    EOF
+    apply_manifest_on(hocon_host, hocon_manifest,
+                      {:acceptable_exit_codes => [0,2]})    
+  end
+
+  def delete_one_hocon_setting(hocon_host,
+                               hocon_file_path, 
+                               hocon_setting)
+    hocon_manifest =<<-EOF
+      hocon_setting { "#{hocon_setting}" :
+        ensure  => absent,
+        path => "#{hocon_file_path}",
+        setting => "#{hocon_setting}",
+      }
+    EOF
+    apply_manifest_on(hocon_host, hocon_manifest,
+                      {:acceptable_exit_codes => [0,2]})    
```

Comment:
Right. I think it is a matter of indifference if it was applied or not, provided the end result is that the setting is set, which both 0 and 2 indicate. It might one day be enabled by default, for example. Or another test may have been sloppy and left state behind. We want to assign a given value to a given key. The output of the `apply` will show what happens, so we're not even losing any information.